src/battery_manager_laptop.cpp

```diff
@@ -0,0 +1,66 @@
+#include <ros/ros.h>
+#include <sensor_msgs/BatteryState.h>
+#include <std_msgs/String.h>
+
+int percentage;
+
+void batteryCallback(const sensor_msgs::BatteryState::ConstPtr& msg)
+{
+    double capacity;
+    double charge;
+
+    if (msg->location == "hero2")
+    {
+        capacity = msg->capacity;
+        charge = msg->charge;
+        percentage = (charge/capacity)*100;
+    }
+}
+
+int main(int argc, char **argv)
+{
+    ros::init(argc, argv, "Battery_manager_laptop");
+    ros::NodeHandle gn;
+
+    int middle = 50;
```

Comment:
Maybe make these rosparams and use this as defaults

Reply:
These threshold values would also need to be global parameters and read only at the starting of the node.